src/Evolution/Systems/GeneralizedHarmonic/TimeDerivative.cpp

```diff
@@ -268,19 +267,6 @@ void TimeDerivative<Dim>::apply(
     for (size_t nu = 0; nu < Dim + 1; ++nu) {
       gauge_constraint->get(nu) += gauge_function->get(nu);
     }
-  } else {
-#ifdef SPECTRE_DEBUG
-    get(*sqrt_det_spatial_metric) =
```

Comment:
I think this block should be removed in the next commit, not this one. (this commit, at least from the name, doesn't sound like it should change functionality)

Reply:
I just looked at the parent code and it's different than I expected. I'll push a commit that does a couple things or drop this change. I was expecting the parent code in `ComputeTimeDerivative.hpp` to set the default value to NaN, but it doesn't right now, it just leaves it uninitialized.